shieldlib/src/shield.ts

```diff
@@ -14,6 +14,8 @@ import {
 import { TextPlacement } from "./shield_text";
 import { StyleImage } from "maplibre-gl";
 
+const narrowCharacters = /[1Iil ]/g;
```

Comment:
Also J, since we’re using the slender version without a top bar.

Some punctuation characters are also pretty narrow.